core/src/main/scala/spire/math/Complex.scala

```diff
@@ -117,22 +117,22 @@ final case class Complex[@sp(Float, Double) T](real: T, imag: T)
    *
    * `sgn(z) = z / abs(z) = abs(z) / z`
    */
-  def complexSignum(implicit f: Field[T], n: NRoot[T], s: Signed[T]): Complex[T] =
+  def complexSignum(implicit f: Field[T], n: NRoot[T], o: Order[T], s: Signed[T]): Complex[T] =
```

Comment:
Why is `Order` required now?

Reply:
See https://github.com/typelevel/spire/pull/1115#issue-790147766 about `Order`.